src/Mod/PartDesign/App/Body.cpp

```diff
@@ -41,18 +41,11 @@ using namespace PartDesign;
 
 PROPERTY_SOURCE(PartDesign::Body, Part::BodyBase)
 
-Body::Body() {
+Body::Body()
+{
     ADD_PROPERTY_TYPE(AllowCompound, (false), "Experimental", App::Prop_None, "Allow multiple solids in Body (experimental)");

```

Comment:
shouldn't this have been changed to true by default?

Reply:
It is changed in the command and that default should be probably preserved if someone recorded macro with it.

Reply:
We want multi body to be default behavior going forward, this should be true when scripting too IMO, if somebody wants the restrictions back it should be an explicit action